helm/cluster-aws/templates/aws-ebs-csi-driver-helmrelease.yaml

```
@@ -32,10 +42,11 @@ spec:
   install:
     remediation:
       retries: 30
-  {{- if .Values.global.connectivity.proxy.enabled }}
-  values:
-    proxy:
-      noProxy: "{{ include "noProxyList" $ }}"
-      http: {{ .Values.global.connectivity.proxy.httpProxy | quote }}
-      https: {{ .Values.global.connectivity.proxy.httpsProxy | quote }}
-  {{ end }}
+  {{- $awsEbsCsiDriverHelmValues := (include "defaultAwsEbsCsiDriverHelmValues" .) | fromYaml -}}
+  {{- $customAwsEbsCsiDriverHelmValues := ($.Values.global.apps).awsEbsCsiDriver -}}
+  {{- if $customAwsEbsCsiDriverHelmValues }}
+  {{- $awsEbsCsiDriverHelmValues = merge (deepCopy $customAwsEbsCsiDriverHelmValues) $awsEbsCsiDriverHelmValues -}}
+  {{- end }}
+  {{- if $awsEbsCsiDriverHelmValues }}
+  values: {{ $awsEbsCsiDriverHelmValues | toYaml | nindent 4 }}
```

```suggestion
  values: {{- $awsEbsCsiDriverHelmValues | toYaml | nindent 4 }}
```

to avoid trailing whitespace

[repeated]